pkg/image/resolve.go

```diff
@@ -349,19 +349,16 @@ func getSourcesList(imageSources map[string]bool) string {
 	return strings.Join(sources, ",")
 }
 
-func CreateCatalogImageListConfigMap(cm *v1.ConfigMap, catalog *v3.Catalog) (err error) {
```

Comment:
This function is no longer used by Rancher, but it is a public facing API. Should I leave this function behind in case someone is using Rancher as a library? Not sure what our policy is with deprecating such exported functions.

Reply:
If we aren't using it, I would prefer to remove it since we already export GetImages() and this package can use some cleanup.

Reply:
cc: @kinarashah what do you think?

Reply:
`CreateCatalogImageListConfigMap` was introduced very recently by https://github.com/rancher/rancher/pull/33151 for 2.6. I don't think this has been released yet, was this elsewhere?

Reply:
Ah yes I think you are right; we should be safe to remove it then. Thanks @kinarashah @PennyScissors!